scss/mixins/_pagination.scss

```diff
@@ -11,17 +11,63 @@
 
 // scss-docs-start pagination-max-items-mixin
 // Boosted mod
-// Ensure we only display `n` items:
-// - the first `n/2` items
-// - the last `n/2` items
-// - including the .active one (so maybe not the `n/2`th last item)
+// The pagination list has (n) items and is built in 3 parts:
+// - the begining of the list is made of the first (n/2) items
+// - the end of the list is made of the first (n/2) items
```

Comment:
If the the begining of the list is made of the first (n/2) items then the end of the list shouldn't be made of the **last** (n/2) items?